docs/modules/releasenotes/pages/whatsnew.adoc

```diff
@@ -50,6 +55,40 @@ You can continue to use the filter as-is; no changes are needed to use this fore
 
 The filter that depends on the `R` package will be discontinued in a future release.
 
+=== Enhanced Linkd Improvements
+
+Enlinkd has been refactored, improving performance, fixing bugs, and adding a number of new features, most notably new Karaf command-line tools:
+
+* enlinkd-discovery-bridge-domain: triggers bridge topology discovery
+* enlinkd-run-collection: triggers an SNMP linkd collection
+* enlinkd-reload: restarts Enlinkd
+* enlinkd-reload-config: reload the Enlinkd configuration
+* enlinkd-reload-topology: refresh topology to force an update based on enlinkd data
+* enlinkd-snmp-collect: perform an Enlinkd SNMP group/table collection
+
+=== Poll Status History Enabled By Default
+
+Our service monitor infrastructure has had support for storing the up/down history of poll status, but it is now enabled by default, including graph definitions for all included services.
+This allows you to get historical availability per-service beyond 24 hours.
+
+=== Experimental Asynchronous Polling
```

Comment:
```suggestion
=== Experimental asynchronous polling
```

Reply:
these always throw me off... they're "headers" so my brain assumes they should be Capitalized, hehe